apex-windows/Cargo.toml

```diff
@@ -14,5 +14,5 @@ futures-core = "0.3.17"
 futures-util = "0.3.17"
 tokio = { version = "1.14.0", features = ["time"] }
 [target.'cfg(target_os = "windows")'.dependencies]
-windows = { version = "0.43", features = ["Media_Control", "Foundation", "std"] }
```

Comment:
I don't currently have a Windows machine to test this but Windows support is on a best effort basis anyway so I'm fine with merging this.

Reply:
i can't test too. that feature deleted at 0.43 version, probably std included default